src/compute/arithmetics/basic/add.rs

```diff
@@ -42,6 +43,34 @@ where
     binary(lhs, rhs, lhs.data_type().clone(), |a, b| a + b)
 }
 
+/// Wrapping addition of two primitive arrays.
+/// It do nothing if the result overflows.
```

Comment:
```suggestion
/// It wraps around at the boundary of the type if the result overflows.
```

(following [the language used in std](https://doc.rust-lang.org/std/primitive.u32.html#method.wrapping_add)